homeassistant/components/tessie/coordinator.py

```diff
@@ -0,0 +1,51 @@
+"""Tessie Data Coordinator."""
+from datetime import timedelta
+from http import HTTPStatus
+import logging
+
+from aiohttp import ClientResponseError, ClientSession
+from tessie_api import get_state_of_all_vehicles
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+TESSIE_SYNC_INTERVAL = 15
```

Comment:
Maybe we rip this out to a config - Up to you whether we do it in this version of future versions

Reply:
I am completely open to doing this, but I think I've had push back on this sort of thing in the past as the user can disable refreshes and schedule whatever they want using automations. For example, I may actually create an automation to increase this to something like 5sec when the car is being driven.

Regardless, I'd want to handle it as a seperate PR so we dont delay this one.

Reply:
Sounds good to me.